pennylane/ops/qutrit/channel.py

```diff
@@ -0,0 +1,216 @@
+    num_wires = 1
+    grad_method = "A"
+    grad_recipe = ([[1, 0, 1], [-1, 0, 0]],)
+
+    def __init__(self, p, wires, id=None):
+        super().__init__(p, wires=wires, id=id)
+
+    @staticmethod
+    def compute_kraus_matrices(p):  # pylint:disable=arguments-differ
+        r"""Kraus matrices representing the depolarizing channel.
+        Args:
+            p (float): each gate is applied with probability :math:`\frac{p}{9}`
+        Returns:
+            list (array): list of Kraus matrices
+
+        **Example**
+
+        >>> np.round(qml.QutritDepolarizingChannel.compute_kraus_matrices(0.5), 3)
+        array([[[ 0.707+0.j   ,  0.   +0.j   ,  0.   +0.j   ],
+           [ 0.   +0.j   ,  0.707+0.j   ,  0.   +0.j   ],
+           [ 0.   +0.j   ,  0.   +0.j   ,  0.707+0.j   ]],
+
+          [[ 0.25 +0.j   ,  0.   +0.j   ,  0.   +0.j   ],
+           [ 0.   +0.j   , -0.125+0.217j,  0.   +0.j   ],
+           [ 0.   +0.j   ,  0.   +0.j   , -0.125-0.217j]],
+
+          [[ 0.25 +0.j   ,  0.   +0.j   ,  0.   +0.j   ],
+           [ 0.   +0.j   , -0.125-0.217j,  0.   +0.j   ],
+           [ 0.   +0.j   ,  0.   +0.j   , -0.125+0.217j]],
+
+          [[ 0.   +0.j   ,  0.25 +0.j   ,  0.   +0.j   ],
+           [ 0.   +0.j   ,  0.   +0.j   ,  0.25 +0.j   ],
+           [ 0.25 +0.j   ,  0.   +0.j   ,  0.   +0.j   ]],
+
+          [[ 0.   +0.j   , -0.125+0.217j,  0.   +0.j   ],
+           [ 0.   +0.j   ,  0.   +0.j   , -0.125-0.217j],
+           [ 0.25 +0.j   ,  0.   +0.j   ,  0.   +0.j   ]],
+
+          [[ 0.   +0.j   , -0.125-0.217j,  0.   +0.j   ],
+           [ 0.   +0.j   ,  0.   +0.j   , -0.125+0.217j],
+           [ 0.25 +0.j   ,  0.   +0.j   ,  0.   +0.j   ]],
+
+          [[ 0.   +0.j   ,  0.   +0.j   ,  0.25 +0.j   ],
+           [ 0.25 +0.j   ,  0.   +0.j   ,  0.   +0.j   ],
+           [ 0.   +0.j   ,  0.25 +0.j   ,  0.   +0.j   ]],
+
+          [[ 0.   +0.j   ,  0.   +0.j   , -0.125-0.217j],
+           [ 0.25 +0.j   ,  0.   +0.j   ,  0.   +0.j   ],
+           [ 0.   +0.j   , -0.125+0.217j,  0.   +0.j   ]],
+
+          [[ 0.   +0.j   ,  0.   +0.j   , -0.125+0.217j],
+           [ 0.25 +0.j   ,  0.   +0.j   ,  0.   +0.j   ],
+           [ 0.   +0.j   , -0.125-0.217j,  0.   +0.j   ]]])
+        """
+        if not math.is_abstract(p) and not 0.0 <= p <= 1.0:
+            raise ValueError("p must be in the interval [0,1]")
+
+        interface = math.get_interface(p)
+        w = math.exp(2j * np.pi / 3)
+        one = 1
+        z = 0
+        if interface == "tensorflow":
+            p = math.cast_like(p, 1j)
+            w = math.cast_like(w, p)
+            one = math.cast_like(one, p)
+            z = math.cast_like(z, p)
+        w2 = w**2
+
+        depolarizing_mats = [
+            [[one, z, z], [z, w, z], [z, z, w2]],
+            [[one, z, z], [z, w2, z], [z, z, w]],
+            [[z, one, z], [z, z, one], [one, z, z]],
+            [[z, w, z], [z, z, w2], [one, z, z]],
+            [[z, w2, z], [z, z, w], [one, z, z]],
```

Comment:
An alternative is to use the X^i Z^j relationship explicitly to evaluate the matrices. This would also ensure their order w.r.t. the definition is preserved.

Reply:
That's really odd - what specifically breaks, and how?

Reply:
First @ operator causes type error, when I do math.matmul, it is just giving different values for the Kraus matrices for the values associated with w and w^2, but it works fine everywhere else.

Reply:
It's fine - recommend to just leave a brief developer note (< 2 lines) that explains why it was done this way.